velox/expression/tests/CustomTypeTest.cpp

```diff
@@ -227,6 +229,8 @@ TEST_F(CustomTypeTest, getCustomTypeNames) {
           "JSON",
           "HYPERLOGLOG",
           "TIMESTAMP WITH TIME ZONE",
+          "UDT2",
+          "MATERIALIZETESTUDT",
```

Comment:
Where are these from?

Reply:
We should not hard-code custom types registered in other tests like this. It is fragile and will break again if another test is added to register some new types.

Reply:
@mbasmanova Can we call `unregisterCustomType` after the use of the custom type?

Reply:
@liujiayi771 I see. Let's update ArrayViewTest and ArrayWriterTest to unregister custom types they use. Let's make these changes in a separate PR though.

Reply:
@mbasmanova Should this current PR add the UDTs they registered to `CustomTypeTest` and then fix it in a separate PR?

Reply:
I suggest to create a PR to fix ArrayViewTest and ArrayWriterTest and land that first. Then rebase this PR and land it.

Reply:
OK. I will do this right now.